$NetBSD$

FL_MINOR_VERSION is defined to 1 in headerfile. Use same value here.
Fix string comparison with "==" to portable "=" sh syntax.
Remove check for libsupc++.

--- configure.in.orig	2012-03-23 05:49:17.000000000 +0000
+++ configure.in
@@ -35,7 +35,7 @@ AC_INIT(src/Widget.cxx)
 
 dnl FLTK library versions...
 FL_MAJOR_VERSION=2
-FL_MINOR_VERSION=0
+FL_MINOR_VERSION=1
 FL_PATCH_VERSION=0
 FL_RELEASE_VERSION=a0
 FL_API_VERSION=${FL_MAJOR_VERSION}.${FL_MINOR_VERSION}
@@ -727,7 +727,7 @@ case $uname in
 	)
 	dnl Check for the Xshm extension unless disabled...
         AC_ARG_ENABLE(xshm, [  --enable-xshm           turn on MIT-SHM support (default=no)])
-	if test x$enable_xshm == xyes; then
+	if test x$enable_xshm = xyes; then
 	  AC_CHECK_HEADER(X11/extensions/XShm.h, AC_DEFINE(USE_XSHM),,
 	    [#include <X11/Xlib.h>])
 	fi
@@ -854,31 +854,6 @@ if test -n "$GCC"; then
     # Previous versions of GCC do not have the reliance on the stdc++
     # or g++ libraries, so the extra supc++ library is not needed.
 
-    AC_MSG_CHECKING(if libsupc++ is required)
-
-    SUPC="`$CXX -print-file-name=libsupc++.a 2>/dev/null`"
-    case "$SUPC" in
-    	libsupc++.a*)
-	    # Library not found, so this is and older GCC...
-	    AC_MSG_RESULT(no)
-	    ;;
-	*)
-            # This is gcc 3.x, and it knows of libsupc++, so we need it
-	case "$uname" in
-	    # do not add for Darwin as it is already linked with it
-	    Darwin*)
-	    ;;
-	    *)
-            LIBS="$LIBS -lsupc++"
-            AC_MSG_RESULT(yes)
-	    ;;
-	esac
-    esac
-
-    if test x$uname != xDarwin; then
-        CXX="$CC"
-    fi
-
     AC_ARG_ENABLE(exceptions, [  --enable-exceptions     turn on GCC exceptions (default=yes)])
     if test x$enable_exceptions != xno; then
         AC_DEFINE(HAVE_EXCEPTIONS)
